lib/ASRouter.jsm

```diff
@@ -73,6 +74,21 @@ const MessageLoaderUtils = {
     return remoteMessages;
   },
 
+  /**
+   * _remoteSettingsLoader - Loads messages for a RemoteSettings provider
+   *
+   * @param {obj} provider An AS router provider
+   * @param {string} provider.bucket The name of the Remote Settings bucket
+   * @returns {Promise} resolves with an array of messages, or an empty array if none could be fetched
+   */
+  async _remoteSettingsLoader(provider) {
+    let messages = [];
+    if (provider.bucket) {
+      messages = await RemoteSettings(provider.bucket).get({filters: {locale: Services.locale.getAppLocaleAsLangTag()}});
```

Comment:
let's add a try/catch to this

Reply:
RemoteSettings already wraps all the exceptions in its `get()`. It returns an empty array on any error.

Reply:
There might be an error on the bucket constructor maybe? I just think it's safer to not trust it